src/lib/Components/Bidding/Screens/Registration.tsx

```diff
@@ -143,17 +155,34 @@ export class Registration extends React.Component<RegistrationProps, Registratio
 
   createBidder() {
     commitMutation(this.props.relay.environment, {
-      onCompleted: () => {
-        console.log("COMPLETED!")
-      },
-      onError: () => {
-        console.log("ERROR!")
-      },
+      onCompleted: (results, errors) =>
+        isEmpty(errors) ? console.log("COMPLETED!", results) : this.presentErrorResult(errors, msgForGraohQLError),
```

Comment:
I left the `console.log` so we can incrementally make changes and work on https://artsyproduct.atlassian.net/browse/PURCHASE-203.